src/components/app.js

```diff
@@ -157,13 +163,24 @@ const noUserManager = { instance: null, error: null };
 
 const STUDY_VIEWS = [StudyView.MAP, StudyView.SPREADSHEET, StudyView.RESULTS];
 
-const App = () => {
+const supportedLanguages = [FRENCH, ENGLISH];
+
+const getSystemLanguage = () => {
```

Comment:
rename to getComputedLanguage or something

Reply:
This method is really to get the language of the system. There's already a method named getComputedLanguage wheter it is a 'sys' code or a language code ('en', 'fr', etc)